index.html

```diff
@@ -42,3 +42,110 @@ <h1 class="hero-title">
+                <li class="Lang-tag">Css</li>
+                <li class="Lang-tag">Javascript</li>
+                <li class="Lang-tag">Html</li>
+            </ul> 
+            <div class="action">
+                <button class="btn-primary primary-font-style">See project</button>
+            </div>
+        </div>
+        </div>
+        
+       <div class="careers-card">
+        <img src="Images/img-placeholder.png" alt="placeholder">
+        <div class="careers-card-footer">
+            <h3 class="careers_title">Multi-Post Stories Gain+Glory</h3>
+            <ul class="tags">
+                <li class="Lang-tag">Ruby on rails</li>
+                <li class="Lang-tag">Css</li>
+                <li class="Lang-tag">Javascript</li>
+                <li class="Lang-tag">Html</li>
+            </ul> 
+            <div class="action">
+                <button class="btn-primary primary-font-style">See project</button>
+            </div>
+        </div>
+        </div>
+
+       <div class="careers-card">
+        <img src="Images/img-placeholder.png" alt="placeholder">
+        <div class="careers-card-footer">
+            <h3 class="careers_title">Multi-Post Stories Gain+Glory</h3>
+            <ul class="tags">
+                <li class="Lang-tag">Ruby on rails</li>
+                <li class="Lang-tag">Css</li>
+                <li class="Lang-tag">Javascript</li>
+                <li class="Lang-tag">Html</li>
+            </ul> 
+            <div class="action">
+                <button class="btn-primary primary-font-style">See project</button>
+            </div>
+        </div>
+        </div>
+
+        <div class="careers-card">
+        <img src="Images/img-placeholder.png" alt="placeholder">
+        <div class="careers-card-footer">
+            <h3 class="careers_title">Multi-Post Stories Gain+Glory</h3>
+            <ul class="tags">
+                <li class="Lang-tag">Ruby on rails</li>
+                <li class="Lang-tag">Css</li>
+                <li class="Lang-tag">Javascript</li>
+                <li class="Lang-tag">Html</li>
+            </ul> 
+            <div class="action">
+                <button class="btn-primary primary-font-style">See project</button>
+            </div>
+        </div>
+        </div>
+
+        <div class="careers-card">
+        <img src="Images/img-placeholder.png" alt="placeholder">
+        <div class="careers-card-footer">
+            <h3 class="careers_title">Multi-Post Stories Gain+Glory</h3>
+            <ul class="tags">
+                <li class="Lang-tag">Ruby on rails</li>
+                <li class="Lang-tag">Css</li>
+                <li class="Lang-tag">Javascript</li>
+                <li class="Lang-tag">Html</li>
+            </ul> 
+            <div class="action">
+                <button class="btn-primary primary-font-style">See project</button>
+            </div>
+        </div>
+        </div>
+    </div>
+</section>
```

Comment:
Great job @187jjay187 🤝

- [ ] As stated in the project requirements, kindly add the `About Me` section to your design.

Reply:
Good day @1GORDON could you please give me some more clarity as to what it is you mean? as I've went over my code and see that i've already added the about me section to my design and have separated it with section dividers, do you perhaps mean the about me section should be an image (e.g. <img src="about-me.png" alt="about">) implemented into the code rather than it being written interlay in code e.g <section id="about_me" class="center-content"> <p>about me</p>